micronaut-camunda-bpm-example/build.gradle

```diff
@@ -6,23 +6,42 @@ plugins {
 group "info.novatec"
 
 micronaut {
-    runtime "netty"
+    // runtime "netty" Commented for JETTY EXAMPLE
+    runtime "jetty"
     testRuntime "junit5"
     processing {
         incremental true
         annotations "micronaut.camunda.bpm.example.*"
     }
 }
-
 dependencies {
     implementation(project(":micronaut-camunda-bpm-feature"))
     runtimeOnly("com.h2database:h2")
     runtimeOnly("ch.qos.logback:logback-classic")
 
     // Test
     testImplementation("io.micronaut:micronaut-http-client")
-}
 
+    // REST API REQUIREMENT
```

Comment:
Please clean up dependencies:
* use syntax as used above
* remove version numbers if possible, i.e. for managed dependencies
* use `$camundaVersion` instead of 7.14
* remove commented out dependency (if not needed)

Reply:
done